_posts/2017/2017-08-08-why-you-should-spend-time-on-your-translation-keys.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Why you should spend time on your translation keys
+date: "2017-08-08 09:34:02"
+image: blog-header/translation-keys.jpg
+categories: tech-stories
+authors: ["Christina"]
+---
+Translation keys are not important at all?
```

Comment:
Suggestion:

Let me guess. You believe that translation keys are not important at all? That taking a closer look at them is just a waste of time? Let me convince you otherwise.